modules/@apostrophecms/page/ui/apos/logic/AposPagesManager.js

```diff
@@ -143,15 +132,24 @@ export default {
       return this.moduleOptions.canCreate;
     }
   },
-  watch: {
-    async pageSetMenuSelection() {
-      await this.getPages();
-    }
+  created() {
```

Comment:
the reason for all this refactoring is to be more aligned with what we do in the piece manager.